README.md

```diff
@@ -89,3 +90,23 @@ type Session = {
 ```
 
 Typically, the service provided by [`fusion-plugin-jwt`](https://github.com/fusionjs/fusion-plugin-jwt) or a custom wrapper around similar key-value store APIs (such as [Redis](https://redis.io/)).
+
+#### `CacheToken`
+
+```js
+import {CacheToken} from 'fusion-tokens';
+```
+
+A token for a caching implementation.
+
+##### Types
+
+```flow
+type Cache = {
+  get(key: string): Promise<any>,
+  set(key: string, val: any): Promise<any>,
+  del(key: string): Promise<any>,
+}
+```
+
+Standard API for caching.
```

Comment:
Is there any current plugins/libraries that we can document as suggestions here?